utils/joiner/tuplejoiner.cpp

```diff
@@ -41,11 +41,10 @@ namespace joiner
 constexpr const size_t DEFAULT_BUCKET_COUNT = 10;
 
 template <typename HashTable>
-std::unique_ptr<HashTable> makeHashMap(size_t bucketCount, ResourceManager* resourceManager)
+std::unique_ptr<HashTable> makeHashMap(size_t /*bucketCount*/, ResourceManager* /*resourceManager*/)
 {
-  return std::unique_ptr<HashTable>(new HashTable(bucketCount, TupleJoiner::hasher(),
-                                                  typename HashTable::key_equal(),
-                                                  utils::STLPoolAllocator<typename HashTable::value_type>(resourceManager)));
+  // boost::unordered_flat_map doesn't need bucket_count or allocator in constructor
```

Comment:
`std::ordered_map` also doesn't need it, but we use custom allocator to track memory usage